src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+
+const convertToObject = (stylesString) => {
+  const styleObject = {};
+
+  const declarations = stylesString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy of the array before operating on it. The 'split' method already produces a new array, so you can chain other methods directly to it.